sensor_array/analysis/sample_analysis.py

```diff
@@ -425,10 +415,10 @@ def execute_sample_analysis(config_file):
     for sample_type in sample_types:
 
         # ----- Load Breath Samples -----
-        _, all_breath_samples_joined = load_breath_samples_alt(breath_samples_filepath)
+        all_breath_samples = load_breath_samples(breath_samples_filepath)
 
         # ========== Limit Breath Sample Range for Testing ==========
-        all_breath_samples_joined = all_breath_samples_joined[0:num_samples_to_test]
+        all_breath_samples = all_breath_samples.loc[0:num_samples_to_test-1]
```

Comment:
do you need loc here?

Reply:
I do need loc here. Without it tries to find keys for 0:num_samples_to_test-1 rather than accessing by index.